### PR TITLE
Add Reconnect button to pigeon GUI

### PR DESCRIPTION
**Pull request by ritishpentapalli**
Add button to reconnect to shepard in pigeon's GUI. Pigeon can now reconnect to shepard mid-flight

### CODE REVIEW

File: pigeon/ui/areas/controlsarea.py [hunk 1]
```diff
@@ -53,6 +54,9 @@ def __init__(self, *args, **kwargs):
         self.last_message_received_time = None
         self.uav_pictures_taken = ""
         self.uav_pictures_transmitted = ""
+        self.reconnectBtn = QtWidgets.QPushButton("Reonnect")
```
Comment (PossiblyAShrub) [inline pigeon/ui/areas/controlsarea.py]: Just a small typo. I think your meant "Reconnect".

**Review by PossiblyAShrub — CHANGES_REQUESTED** [review]
Looks good so far, thanks! We should disable or hide the "reconnect" button if we are currently connected though.

File: pigeon/ui/areas/controlsarea.py (hunk 1 above)
Comment (ritishpentapalli) [reply pigeon/ui/areas/controlsarea.py]: > Just a small typo. I think your meant "Reconnect".

Changed button name to "Reconnect".

**Comment by ritishpentapalli** [comment]
> Looks good so far, thanks! We should disable or hide the "reconnect" button if we are currently connected though.

Implemented feature.

**Review by PossiblyAShrub — APPROVED** [review]
Thanks!

**Pull request merged**